notebook/static/notebook/js/menubar.js

```diff
@@ -130,6 +130,158 @@ define([
         this._new_window(url);
     };
 
+    MenuBar.prototype._nbconvert_upload_conf = function(download) {
+      var body = $("<div>");
+      var notebook_path = utils.encode_uri_components(
+        this.notebook.notebook_path
+      );
+
+      var build_json_for_post = function(notebook, config, format) {
+        var json_to_pass = {
+          notebook_contents: {
+            content: notebook.toJSON(),
+            name: notebook.notebook_name,
+            last_modified: notebook.last_modified
+        },
+          export_format: format,
+          config: config
+        };
+        return json_to_pass;
+      };
+
+      var form = $("<form>");
+      var fileinput = $("<input>")
+        .attr("type", "file")
+        .attr("tabindex", "1");
+
+      var fileformat = $("<select>");
+
+      var export_opts = {
```

Comment:
I don't see `slides` here... could be the case?

Reply:
Well the tests pass after I made the changes @rgbkrk suggested, but I definitely should include slides there. Apologies!